scripts/jenkins/cloud/ansible/roles/supportconfig/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
same here
also, in some files you add a blank line before `---` and others you don't, could you keep the same for all files?